content/this-month/2022-01/index.md

```diff
@@ -37,6 +37,12 @@ The `x86_64` crate provides various abstractions for `x86_64` systems, including
 
 In January, …
 
+### [`multiboot2`](https://github.com/rust-osdev/multiboot2)
+
+The `multiboot2` crate provides abstraction types for the boot information of multiboot2 bootloaders.
+The latest release of the `multiboot2`-crate is now `v.0.13.1` (was `v0.12.2`). It contains
+minor improvements. For example, missing getters were added.
```

Comment:
Perhaps we could link to commit https://github.com/rust-osdev/multiboot2/commit/155698123d15afd3dd44994f7bb85976ede21afd here to provide more details for interested readers?